File: BackEnd/ebanking/src/main/java/com/ebanking/dto/TransactionRequestDTO.java
package com.ebanking.dto;

import java.util.Date;

public class TransactionRequestDTO {
	private String cardNumber;
	private Date from;
	private Date to;
	private int maxResult;
	public String getCardNumber() {
		return cardNumber;
	}
	public void setCardNumber(String cardNumber) {
		this.cardNumber = cardNumber;
	}
	public Date getFrom() {
		return from;
	}
	public void setFrom(Date from) {
		this.from = from;
	}
	public Date getTo() {
		return to;
	}
	public void setTo(Date to) {
		this.to = to;
	}
	public int getMaxResult() {
		return maxResult;
	}
	public void setMaxResult(int maxResult) {
		this.maxResult = maxResult;
	}
	
	
}

File: FrontEnd/ebankingui/src/app/dtos/TransactionRequestDTO.ts
export class TransactionRequestDTO {
    cardNumber: string;
	from: Date;
	to: Date;
	maxResult: number;
	constructor(cardNumber: string, from: Date, to: Date, maxResult: number) {
		this.cardNumber = cardNumber === null ? '' : cardNumber;
		this.from = from === null ? null: from;
		this.to = to === null ? null : to;
		this.maxResult = maxResult === null ? 0 : maxResult;

	}
}